packages/frontend/gtag/src/gtag-ga4-events/gtag-ga4-event-share.ts

```diff
@@ -0,0 +1,22 @@
+import { gtagEvent } from "../gtag-event";
+
+export enum GtagGa4EventShareMethod {
+  TWITTER = "Twitter",
```

Comment:
May as well add all major social media options as an enum ready for use?

Reply:
Will do - More can be added if required